mindsdb/libs/helpers/log.py

```diff
@@ -0,0 +1,92 @@
+import pprint
+import logging
+import time
+from logging import handlers
+import colorlog
+from inspect import getframeinfo, stack
+import socketio
+
+global internal_logger
+global id
+global sio
+
+
+
+def gen_char(n, ch):
```

Comment:
agreed, will add some comments in a sec... and also fix the naming of the function & args.

Reply:
I added some comments where I thought it was appropriate, I don't want to go overboard with commenting either, because that just results in comments going out of sync with the code.